src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/EntityDtoValidatorTest.java

```diff
@@ -386,4 +422,15 @@ private void assertError(String fieldName, String message, Errors errors) {
     private String getQualifiedFieldName(String field) {
         return ENTITY_FIELD + "." + field;
     }
+
+    private AddressDto getValidAddressDto() {
```

Comment:
We have got already mocked [here](https://github.com/companieshouse/overseas-entities-api/blob/3da36724f57a3094e6f4cadd4c828dcf22ad06e3/src/test/java/uk/gov/companieshouse/overseasentitiesapi/mocks/AddressMock.java#L16)

Reply:
Oh yes! Thanks, will use that.